README.md

```diff
@@ -144,21 +144,72 @@ Afterwards, when you run your environment e.g. with `docker-compose up stablex`
+The binary can be configured via command line options and environment variables: `cargo run -- --help`
+
+```
+Gnosis Exchange protocol driver.
+
+USAGE:
+    driver [OPTIONS] --network-id <network-id> --node-url <node-url> --private-key <private-key>
+
+FLAGS:
+    -h, --help
+            Prints help information
+
+    -V, --version
+            Prints version information
+
+
+OPTIONS:
+        --auction-data-page-size <auction-data-page-size>
+            The page size with which to read orders from the smart contract [env: AUCTION_DATA_PAGE_SIZE=]  [default:
+            100]
+        --http-timeout <http-timeout>
+            The default timeout in milliseconds of HTTP requests to remote services such as the Gnosis Safe gas station
+            and exchange REST APIs for fetching price estimates [env: HTTP_TIMEOUT=]  [default: 10000]
+        --log-filter <log-filter>
+            The log fiter to use.
+
+            This follows the `slog-envlogger` syntax (e.g. 'info,driver=debug'). [env: DFUSION_LOG=]  [default: info]
+    -i, --network-id <network-id>
+            The network ID used for signing transactions (e.g. 1 for mainnet, 4 for rinkeby, 5777 for ganache) [env:
+            NETWORK_ID=]
+    -n, --node-url <node-url>
+            The Ethereum node URL to connect to. Make sure that the node allows for queries witout a gas limit to be
+            able to fetch the orderbook [env: ETHEREUM_NODE_URL=]
+        --orderbook-filter <orderbook-filter>
+            JSON encoded object of which tokens/orders to ignore.
+
+            For example: '{ "tokens": [1, 2], "users": { "0x7b60655Ca240AC6c76dD29c13C45BEd969Ee6F0A": { "OrderIds": [0,
+            1] }, "0x7b60655Ca240AC6c76dD29c13C45BEd969Ee6F0B": "All" } }' [env: ORDERBOOK_FILTER=]  [default: {}]
+        --price-source-update-interval <price-source-update-interval>
+            Time interval in seconds in which price sources should be updated [env: PRICE_SOURCE_UPDATE_INTERVAL=]
+            [default: 300]
+    -k, --private-key <private-key>
+            The private key used by the driver to sign transactions [env: PRIVATE_KEY]
+
+        --rpc-timeout <rpc-timeout>
+            The timeout in milliseconds of web3 JSON RPC calls, defaults to 10000ms [env: WEB3_RPC_TIMEOUT=]  [default:
+            10000]
+        --scheduler <scheduler>
+            The kind of scheduler to use [env: SCHEDULER=]  [default: system]
+
+        --solver-time-limit <solver-time-limit>
+            The offset from the start of the batch to cap the solver's execution time [env: SOLVER_TIME_LIMIT=]
+            [default: 210]
+        --solver-type <solver-type>
+            Which style of solver to use. Can be one of: 'NAIVE' for the naive solver; 'MIP' for mixed integer
+            programming solver; 'NLP' for non-linear programming solver [env: SOLVER_TYPE=]  [default: naive-solver]
+        --target-start-solve-time <target-start-solve-time>
```

Comment:
Why does everything repeat itself `--thing <thing>`?

Reply:
I think that is the default behaviour of the automatically generated help when using `StructOpt` for generating command-line and environment variable parsing code.

We can use `name = NAME` to manually specify the name of the variable (so that name that shows up between angle brackets, `<NAME>`). The default behaviour is to use the field name.

Reply:
Would be a nice and easy cleanup PR :)

Reply:
> Would be a nice and easy cleanup PR :)

#670